schemas/telemetry/pioneer-study/pioneer-study.1.parquetmr.txt

```diff
@@ -0,0 +1,20 @@
+message pioneer-study {
+  required binary creationDate (UTF8);
+  required binary id (UTF8);
+  required binary type (UTF8);
+  required double version;
+  required group payload {
+    required binary encryptedData (UTF8);
```

Comment:
I thought the original point of this exercise was to not make each consumer decrypt the data?